include/evmc/loader.h

```diff
@@ -77,28 +77,36 @@ enum evmc_loader_error_code
  * It is safe to call this function with the same filename argument multiple times
  * (the DLL is not going to be loaded multiple times).
  *
- * @param filename    The null terminated path (absolute or relative) to the shared library
- *                    containing the EVM implementation. If the value is NULL, an empty C-string
- *                    or longer than the path maximum length the ::EVMC_LOADER_INVALID_ARGUMENT is
- *                    signaled.
+ * @param filename    The null terminated path (absolute or relative) to an EVMC module
+ *                    (dynamically loaded library) containing the VM implementation.
+ *                    If the value is NULL, an empty C-string or longer than the path maximum length
+ *                    the ::EVMC_LOADER_INVALID_ARGUMENT is signaled.
  * @param error_code  The pointer to the error code. If not NULL the value is set to
  *                    ::EVMC_LOADER_SUCCESS on success or any other error code as described above.
  * @return            The pointer to the EVM create function or NULL.
  */
 evmc_create_fn evmc_load(const char* filename, enum evmc_loader_error_code* error_code);
 
 /**
- * Dynamically loads the VM DLL and creates the VM instance.
+ * Dynamically loads the EVMC module and creates the VM instance.
  *
  * This is a macro for creating the VM instance with the function returned from evmc_load().
  * The function signals the same errors as evmc_load() and additionally:
- *  - ::EVMC_LOADER_INSTANCE_CREATION_FAILURE when the create function returns NULL,
- *  - ::EVMC_LOADER_ABI_VERSION_MISMATCH when the created VM instance has ABI version different
- *  from the ABI version of this library (::EVMC_ABI_VERSION).
+ * - ::EVMC_LOADER_INSTANCE_CREATION_FAILURE when the create function returns NULL,
+ * - ::EVMC_LOADER_ABI_VERSION_MISMATCH when the created VM instance has ABI version different
+ *   from the ABI version of this library (::EVMC_ABI_VERSION).
  *
  * It is safe to call this function with the same filename argument multiple times:
  * the DLL is not going to be loaded multiple times, but the function will return new VM instance
  * each time.
+ *
+ * @param filename    The null terminated path (absolute or relative) to an EVMC module
+ *                    (dynamically loaded library) containing the VM implementation.
+ *                    If the value is NULL, an empty C-string or longer than the path maximum length
+ *                    the ::EVMC_LOADER_INVALID_ARGUMENT is signaled.
+ * @param error_code  The pointer to the error code. If not NULL the value is set to
+ *                    ::EVMC_LOADER_SUCCESS on success or any other error code as described above.
+ * @return            The pointer to the created VM.
```

Comment:
Or `NULL` in case of an error. An `error_code` is supplied in this case.

Reply:
Changed.